go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/buildpacks/lifecycle
 require (
 	github.com/BurntSushi/toml v1.1.0
 	github.com/apex/log v1.9.0
-	github.com/buildpacks/imgutil v0.0.0-20220310160537-4dd8bc60eaff
+	github.com/buildpacks/imgutil v0.0.0-20220503154324-ea435be1d82b
```

Comment:
Which commit is this pointing to? How is it working if the changes have not been merged yet?

Reply:
It's pointing to the imgutil branch commit ;)

Reply:
Ah - I just merged the imgutil PR. Could you point it to the main commit?

Reply:
Yeap, let me do it